### ISSUE TITLE
Helper method te() should return an HtmlString and should always escape every parameters

### ISSUE THREAD

**Issue by pgrenaud** [eXolnet/laravel-translation-editor#15]
We should avoid using `{!! te('key') !!}` everywhere and instead always return an HtmlString. We should also always escape all parameters passed in the `$replace` argument (second argument of the `get()` method) instead of doing `{!! te('key', ['a' => e($input)]) !!}` everywhere.

**Comment by xel1045** [comment]
Indeed, `{{ te('key') }}` should be used and it's already the supported: https://github.com/eXolnet/laravel-translation-editor/blob/584e302ff3231f1e012c0f83b478cfbf53075089/src/TranslationEditor.php#L101

**Comment by xel1045** [comment]
Indeed, `{{ te('key') }}` should be used and it's already the supported: https://github.com/eXolnet/laravel-translation-editor/blob/584e302ff3231f1e012c0f83b478cfbf53075089/src/TranslationEditor.php#L101

For the second feature request (i.e. escaping `$replace`), I don't think it's a good idea. Since we're only forwarding translations to Laravel's translation getter, I think it's better if we keep `trans()` and `te()` aligned.

**Comment by pgrenaud** [comment]
As discussed, `getEditor()` does return an HtmlString. But, this method is only called when the editor is enabled. When the editor is disabled, `getTranslation()` is directly called and this method does not return an HtmlString. You did propose to build a blade component to fix this specific issue, but I do not remember exactly how this will resolve it.
